nextcloudappstore/core/templates/app/detail.html

```diff
@@ -110,5 +103,33 @@ <h4 id="downloads">{% trans "Downloads" %}</h4>
             </section>
         </div>
     </div>
+    {% if object.issue_tracker or object.discussion or request.user.is_authenticated %}
+        <div class="row app-comments">
+            <ul class="col-md-12">
+                <h4 class="section-heading" id="comments">{% trans 'Comments' %}</h4>
```

Comment:
Feels like this section is broader than just "Comments". I can't come up with a better alternative, though.